packages/node/src/plugins/StreamingTargetPlugin.ts

```diff
@@ -19,20 +19,16 @@ class StreamingTargetPlugin {
   }
 
   apply(compiler: Compiler) {
-    if (compiler.options.target) {
-      console.warn(
-        `target should be set to false while using NodeSoftwareStreamRuntime plugin, actual target: ${compiler.options.target}`
-      );
-      console.info('Setting target to false');
-      compiler.options.target = false;
```

Comment:
So we can use target 'node' now, and use the default node configs . Have you proved it can works in nextjs or pure webpack project succeed?

Reply:
Its working in next.js, if it were to break, it usually is next that does. I will backtest this in pure webpack/node codebases that i have to verify. But seems to work